mod30/README.md

```diff
@@ -58,15 +77,16 @@ code.
 |[4 - Social Media Integration with Logic Apps ](demos.md#demo-4-social-media-integration-with-logic-apps)|[Link](https://globaleventcdn.blob.core.windows.net/assets/mod/mod30/MOD30_Demo4.mp4)
 |[5 - Automatic Image Captioning with Cognitive Services](demos.md#demo-4-automatic-image-captioning-with-logic-apps)|[Link](https://globaleventcdn.blob.core.windows.net/assets/mod/mod30/MOD30_Demo5.mp4)
 
-## Become a Presenter
+## Become a Trained Presenter
 
-To become a *Trained Presenter*, contact [scalablecontent@microsoft.com](mailto:scalablecontent@microsoft.com). In your email please include:
+To become a trained presenter, contact [learningpathmanager@microsoft.com](mailto:learningpathmanager@microsoft.com). In your email please include:
```

Comment:
Which email is correct? Did you sync on latest? @lbugnion made some updates today so be sure to sync those first.

Reply:
This was updated to be standardized, [based on the apps10 example](https://github.com/microsoft/ignite-learning-paths-training-apps/tree/master/apps10#become-a-trained-presenter). Assumed this email might be correct one. @FBoucher which email should we be using?

Reply:
I'm going to use the old one for now till this is clarified